qutebrowser/misc/sessions.py

```diff
@@ -327,6 +333,7 @@ def save(self, name, last_window=False, load_next_time=False,
             load_next_time: If set, prepares this session to be load next time.
             only_window: If set, only tabs in the specified window is saved.
             with_private: Include private windows.
+            minimal: Don't save tab history
```

Comment:
hmm, looking at the argument name "minimal" alongside these other ones it seems less descriptive. Can you think of a name that fits better alongside the other arguments and would be less likely to be misinterpreted?

Reply:
`--no-history`maybe a better a better fit but I don't no what would be the flag for this one, `-n` or `-h`